NewSpring/Rock
# Feature: Session Navigation for Devotionals

## DESCRIPTION

This PR adds UI and functionality to help users better understand the context of where they are within a study, and to navigate between devotionals within a study.

### How do I test this PR?
To test this PR, please connect to my development database (see connectionstrings snippet below - password redacted) and verify that the following have been added to the devotional entry pages:

```
<?xml version="1.0"?>
<connectionStrings>
	<add name="RockContext" connectionString="Data Source=rockdev.database.windows.net;Initial Catalog=Brian; User Id=nsrock; password=***********;MultipleActiveResultSets=true" providerName="System.Data.SqlClient"/>
</connectionStrings>
```

- [x] Session number should be visible at the top of devotional entries above the h1 heading
- [x] Session number UI should NOT be visible on articles, news, or story entry pages
- [x] Next/Prev buttons at the bottom of the content card
- [x] Previous should not be visible when viewing the first devotional of a study
- [x] Next should not be visible on the last devotional of a study
- [x] Next/prev should NOT be visible on articles, news, or story entry pages
- [x] Session count should be visible between the next/prev buttons, and accurate (2 of 7, etc)
- [x] Next/Prev buttons should be properly linked to the next/prev devotional in the study
- [x] Child items swiper should be initially set to whichever devotional is currently being viewed - if you're on session 6, session 6 should be the middle "active" slide in the swiper.

![image](https://user-images.githubusercontent.com/2465823/63703171-7298a780-c7f6-11e9-8fc1-f95a873d93f3.png)

![image](https://user-images.githubusercontent.com/2465823/63703219-85ab7780-c7f6-11e9-985f-d1098c246058.png)

## TODO


## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
